src/ossia-max/src/search_filter.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "ext.h"
+#include <ossia/network/base/node.hpp>
+
+#define MAX_NUM_ITEMS 256
+
+namespace ossia {
+namespace max {
+struct search_filter
+{
+  t_symbol* m_filter_types[MAX_NUM_ITEMS];
+  long m_filter_types_size{};
+  t_symbol* m_filter_tags[MAX_NUM_ITEMS];
+  long m_filter_tags_size{};
+  t_symbol* m_filter_modes[MAX_NUM_ITEMS];
```

Comment:
isn't that a lot of items ? 
also, instead of a #define, I'd go for a 

     static const constexpr int MAX_NUM_ITEMS = 256;

inside the search_filter struct, this way it won't conflict with other things if one day someone wants to do the same thing in another source file

Reply:
256 is the maximum number of attribute array item in Max
I put only for mode attribute filter